decentraland/unity-renderer
# feat: skins avatar category

## What does this PR change?

`Closes #1643 `
Adds a new wearable category: `skins`.
The skins hides the rest of the categories of the avatar.
A new category es shown in the avatar selection.
You can wear accessories on top of the skin.
A toast is shown if the item hides or replaces other items respectively.

## How to test the changes?



1. Get a skin wearable in your wallet (in test network you can ask @olavra to send you one)
2. Go to: https://play.decentraland.zone/?renderer-branch=feat/skins&NETWORK=ropsten
3. Open the start menu (pressing TAB) -> Backpack
4. Go to skin category, you should see some wearables there
5. Hover your mouse over a skin, it should show a toast over the item saying "wearable will hide other items"
6. Wear a skin
7. Your avatar should hide all other wearables you had
8. Wear any accessory, like glasses, tiaras, etc (toast should not appear since its not going to replace the skin). They should be rendered on top of the skin
9. Change the category and hover the mouse over any item. A toast should appear saying "wearable will replace other items"
10. Equip the item
11. Your avatar should be seen as it used to be with the new wearable selected
12. Finally go to ORG, you should not see the skin category available in avatar editor (feature flag off)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
